test/integration/test_dropbox.py

```diff
@@ -310,3 +291,48 @@ def test_clone_when_team_linked(self, dbx_team_from_env):
         new_dbxt = dbx_team_from_env.clone()
         assert dbx_team_from_env is not new_dbxt
         assert isinstance(new_dbxt, dbx_team_from_env.__class__)
+
+def test_default_oauth2_urls():
+    flow_obj = DropboxOAuth2Flow('dummy_app_key', 'dummy_app_secret',
+        'http://localhost/dummy', 'dummy_session', 'dbx-auth-csrf-token')
+
+    assert re.match(
+        r'^https://{}/oauth2/authorize\?'.format(re.escape(session.WEB_HOST)),
+        flow_obj._get_authorize_url('http://localhost/redirect', 'state', 'legacy'),
+    )
+
+    assert flow_obj.build_url(
+        '/oauth2/authorize'
+    ) == 'https://{}/oauth2/authorize'.format(session.API_HOST)
+
+    assert flow_obj.build_url(
+        '/oauth2/authorize', host=session.WEB_HOST
+    ) == 'https://{}/oauth2/authorize'.format(session.WEB_HOST)
+
+def test_bad_auth(dbx_session):
+    # Test malformed token
+    malformed_token_dbx = Dropbox(MALFORMED_TOKEN, session=dbx_session)
+    # TODO: backend is no longer returning `BadInputError`
+    # with pytest.raises(BadInputError,) as cm:
+    #     malformed_token_dbx.files_list_folder('')
+    # assert 'token is malformed' in cm.value.message
```

Comment:
Can we remove these comments, or is the intention to fix this?

Reply:
The intention is to fix this in a future PR. This is also not technically introduced in this PR, I've just moved this test out of the class.